src/com/geekhub/hw3/auth/authexceptions/AuthException.java

```diff
@@ -0,0 +1,15 @@
+package com.geekhub.hw3.auth.authexceptions;
```

Comment:
It's enough to name this package "exceptions", because you already are in auth package.